source/includes/streaming-distinction.rst

```diff
@@ -1,3 +1,8 @@
 .. important::
 
-   `Spark Structured Streaming <https://spark.apache.org/docs/latest/structured-streaming-programming-guide.html>`__ and `Spark Streaming with DStreams <https://spark.apache.org/docs/latest/streaming-programming-guide.html>`__ are different. 
+   Apache Spark contains two different stream-processing engines:
+   `Spark Streaming with DStreams <https://spark.apache.org/docs/latest/streaming-programming-guide.html>`__,
+   now an unsupported legacy engine, and
+   `Spark Structured Streaming <https://spark.apache.org/docs/latest/structured-streaming-programming-guide.html>`__.
```

Comment:
S: I think this would be easier to read as a bulleted list:
```suggestion
   Apache Spark contains two different stream-processing engines:
   - `Spark Streaming with DStreams <https://spark.apache.org/docs/latest/streaming-programming-guide.html>`__,
   now an unsupported legacy engine
   - `Spark Structured Streaming <https://spark.apache.org/docs/latest/structured-streaming-programming-guide.html>`__.
```